source/Willow-Playground-Smalltalks2017/PresentationStepWebView.class.st

```diff
@@ -29,24 +29,19 @@ PresentationStepWebView >> description [
 PresentationStepWebView >> renderCarouselControlIn: carouselIdentifier slidingTo: aSlideTarget anchorStyledWith: anAnchorClass spanStyledWith: aSpanClass named: aName on: aCanvas [
 
 	aCanvas anchor
-		classes:
-			{anAnchorClass.
-			'carousel-control'};
+		class: anAnchorClass;
```

Comment:
It seems that a lot of this method could be changed to use the new apply: method with the command builder protocol.